### ISSUE TITLE
Consider adding IRC chat interface to website

### ISSUE THREAD

**Issue by natea** [CodeRaising/coderaising#3]
This django plugin (http://gnotty.jupo.org/irc/) makes it easy to provide a web-based chat interface to any IRC chat channel. might make it easier for the newbies to join in the real-time chat without having to learn all the IRC stuff first.

**Comment by briandant** [comment]
Here's something we've been thinking in the Code Mentors discussion:  Remove the "fear of the channel" by only inviting people who have attended events, or that we know.  Maybe you have been to three events, and then you get an invite to the community.  This also forces in-person relationships, instead of just web-based.  Thoughts?

**Comment by natea** [comment]
I like the idea of inviting people to the channel who you have met in person. This keeps out trolls and also fosters stronger in-person relationships. But I wonder how much overhead this would be to keep track of who has been to an  event?

**Comment by briandant** [comment]
Idea: If users have an account, can we put the responsibility on them to set the number of times they've attended, and that let's them into the chat?  This isn't perfect security, but in a few lines of code we can get 98% of the way to limited access + security.

**Comment by natea** [comment]
yeah, maybe we can make an RSVP feature for mezzanine.events and tie it
their User profile, so we have a record of how many times they attended.
http://www.coderaising.org/events/code-raising-kickoff/

On Fri, Feb 8, 2013 at 7:03 PM, Brian Dant notifications@github.com wrote:

> Idea: If users have an account, can we put the responsibility on them to
> set the number of times they've attended, and that let's them into the
> chat? This isn't perfect security, but in a few lines of code we can get
> 98% of the way to limited access + security.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/CodeRaising/coderaising/issues/3#issuecomment-13321759..

## 

nate@appsembler.com
+1 (617) 517-4953
http://twitter.com/natea | http://linkedin.com/in/natea
